src/components/ChatMessages.tsx

```diff
@@ -1,43 +1,42 @@
-import { reverse } from "dns/promises";
-import React, { useEffect, useRef, useState } from "react";
+import React, { useState } from "react";
 
-import { getData } from "../actions/actions";
+import { useMessages } from "../contexts/Messages";
 import { useScrollToBottom } from "../hooks/useScrollToBottom";
-import { Data } from "./types";
+import ChatTextInput from "./ChatTextInput";
+import { Message } from "./types";
 
 const ChatMessages = () => {
-  const [data, setData] = useState<Data[]>();
+  const [renderData, setRenderData] = useState(0);
```

Comment:
this should be redundant 🤔  `useMessages` should take care of it. Doesn't it works?